B-Sockets-ServerTesting/sockets-servertesting.md

```diff
@@ -262,7 +265,9 @@ are particularly useful, as they allow you to specify the number of concurrent
 command will create 25 concurrent users that will each attempt to hit the server
 50 times, resulting in 1250 hit attempts:
 
-`siege -c 25 -r 50 http://<hostname>:<port>/<URI>`
+```
```

Comment:
nit: console

Reply:
this one shouldn't be console - console is for when you have a prompt or the command's output
It could be `sh` or `bash`, but it wouldn't change much for this one